src/actions/breakpoints/index.js

```diff
@@ -157,15 +157,12 @@ export function toggleBreakpoints(
   breakpoints: BreakpointsMap
 ) {
   return async ({ dispatch }: ThunkArgs) => {
-    const promises = breakpoints
-      .valueSeq()
-      .toJS()
-      .map(
-        ([, breakpoint]) =>
-          shouldDisableBreakpoints
-            ? dispatch(disableBreakpoint(breakpoint.location))
-            : dispatch(enableBreakpoint(breakpoint.location))
-      );
+    const promises = (Object.values(breakpoints): any).map(
```

Comment:
could we use `getBreakpointsArray`?

Reply:
It looks like the code that calls this function is already passing in an array instead of a map so I can just change the type and drop the Object.values() call